content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/_cq_dialog/.content.xml

```diff
@@ -48,6 +48,15 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <label
```

Comment:
This should at the minimum be the last field in the dialog as it's less important than the other options, but I would also consider putting it in a new tab as you have with the other components, even though it contains just one field.

Reply:
Moved the label to a separate tab